scripts/tasks/sandbox-parts.ts

```diff
@@ -350,11 +350,7 @@ function addExtraDependencies({
   debug: boolean;
 }) {
   // web-components doesn't install '@storybook/testing-library' by default
-  const extraDeps = [
-    '@storybook/jest',
-    '@storybook/testing-library@next',
-    '@storybook/test-runner@next',
```

Comment:
This will only work when each of these packages have their `next` branch merged to `main` and released as `latest`.

(releasing the `next`-branch as `latest` was also an option we talked about.